src/include/connection.h

```diff
@@ -253,13 +253,14 @@ struct __wt_connection_impl {
     WT_TXN_GLOBAL txn_global; /* Global transaction state */
 
     WT_RWLOCK hot_backup_lock; /* Hot backup serialization */
-    bool hot_backup;           /* Hot backup in progress */
+    time_t hot_backup_start;   /* Clock value of most recent checkpoint needed by hot backup */
```

Comment:
I had to change the type here since the signature of `__wt_seconds` is different in 4.0.